Automattic/woocommerce-payments
# Fix display of multiple payment method fees

Fixes: #3241
Discussion p1635391455166700-slack-CGGCLBN58
Related PR: #3055 

#### Changes proposed in this Pull Request
- show discounts only on overview page ( do not show the fees if there are no discounts )
- add translations for all available methods
- fix a small conditional logic issue.
- show only payment methods that are added to the particular store
- "Learn more" is be added only 1 time and not for every payment method









#### Testing instructions
- Add a discount fee with multiple payment methods. You can do this in 2 ways:
1. Select a fee structure on server side (wcpay/fees folder) and add it using the CLI command: 
`wp wcpay add_fee_structure_to_account <stripe_account_id> --fee_structure_id=<fee_structure_id>`
2. Add the fee manually in the database,`wcpay_test_account_fee_structures` table. In order to have the fee active `end_date` field should be in the future or null.

We should have only 1 fee structure active for an account. In theory, an account can have more than one account fee structure at the same time. However, the current implementation of promotions only allows a single fee structure to be applied at the point an account is created. So we're unlikely to see this in practice at the moment.

- Re-login or clear the account cache 
- Go to Payments -> Overview
- Check if the discount is displayed properly under the Account details->Active discounts  section for each payment method





-------------------


**Post merge**
